attest/attest.go

```diff
@@ -224,11 +224,16 @@ var (
 
 // TPMInfo contains information about the version & interface
 // of an open TPM.
+// FWVersionMajor and FWVersionMinor describe the firmware version of
```

Comment:
Comment should probably be over the fields not the struct?

Are there any links or background you can add to this comment? E.g. what are reasonable values to take? Could these potentially indicate vulnerable TPMs?

Reply:
Done. Not really, the values are vendor specific.